soh/soh/Enhancements/randomizer/3drando/spoiler_log.hpp

```diff
@@ -8,7 +8,7 @@
 
 using RandomizerHash = std::array<std::string, 5>;
 
-
+// RANDOTODO this is primerally used by the check tracker now, and should probably be moved
```

Comment:
This is used by hook_handlers to determine which flag type to check against when triggered. That field would probably be better off using `FlagType` instead.

Reply:
also, *primarily